lib/OktaAuth.ts

```diff
@@ -537,7 +537,11 @@ class OktaAuth implements SDKInterface, SigninAPI, SignoutAPI {
     if (accessToken && this.tokenManager.hasExpired(accessToken)) {
       accessToken = null;
       if (autoRenew) {
-        accessToken = await this.tokenManager.renew('accessToken') as AccessToken;
+        try {
```

Comment:
Devs may call isAuthenticated independently from auto renew process, at that time, they will expect an error to be caught from the method. 
The error in auto renew process should be handled in the flow instead of the specific method.
See change here: https://github.com/okta/okta-auth-js/pull/1033